prestoadmin/prestoclient.py

```diff
@@ -310,3 +325,21 @@ def _get_private_key(self, keystore):
             error('No alias %s found in %s. Available aliases are %s' %
                   (alias, self.coordinator_config.get_client_keystore_path(),
                    all_keys))
+
+    @staticmethod
+    def create_auth_headers(user, password):
+        if not user:
```

Comment:
eh, I think it's good to keep these generic checks to verify that the LDAP contract is fulfilled